src/main/java/org/jabref/gui/externalfiles/DownloadFullTextAction.java

```diff
@@ -111,10 +111,7 @@ private void downloadFullTexts(Map<BibEntry, Optional<URL>> downloads, BibDataba
             if (result.isPresent()) {
                 Optional<Path> dir = databaseContext.getFirstExistingFileDir(Globals.prefs.getFilePreferences());
                 if (dir.isEmpty()) {
-                    dialogService.showErrorDialogAndWait(Localization.lang("Directory not found"),
-                            Localization.lang("Main file directory not set!") + " "
-                                    + Localization.lang("Preferences")
-                                    + " -> " + Localization.lang("File"));
+                    dialogService.showErrorDialogAndWait(Localization.lang("Main file directory not set. Check the preferences (linked files) or the library properties."));
```

Comment:
I think the first argument "Localization.lang("Directory not found")" should still be there, so that the error dialog gets a nice header.

Reply:
Done! 😀

Reply:
Here is a screenshot! 
<img width="517" alt="Screen Shot 2020-12-30 at 4 23 50 PM" src="https://user-images.githubusercontent.com/33851952/103386899-1e729700-4abe-11eb-904d-0e5e532680d7.png">